packages/compiler-sfc/src/script/utils.ts

```diff
@@ -113,8 +113,8 @@ export const joinPaths = (path.posix || path).join
  * key may contain symbols
  * e.g. onUpdate:modelValue -> "onUpdate:modelValue"
  */
-export const escapeSymbolsRE = /[ !"#$%&'()*+,./:;<=>?@[\\\]^`{|}~]/g
```

Comment:
We can separate it into two regex patterns and remove `g` flag for `getEscapedKey` function.

Reply:
Thanks for the review! I separated them

Reply:
If we have two rules, we should put them together so that if we need to change one rule in the future, we can ensure that the second one is not missed and leads to inconsistency.